src/matrix/layout_info.cpp

```diff
@@ -21,32 +21,35 @@ LayoutInfo::LayoutInfo(const LocalElementSize& size, const TileElementSize& bloc
       tile_offset_row_(tile_offset_row), tile_offset_col_(tile_offset_col) {
   using util::size_t::sum;
 
-  DLAF_ASSERT(size_.isValid(), "Invalid Matrix size!");
-  DLAF_ASSERT(!block_size_.isEmpty(), "Invalid Block size!");
+  DLAF_ASSERT(size_.isValid(), size_);
+  DLAF_ASSERT(!block_size_.isEmpty(), block_size_);
 
   nr_tiles_ = {util::ceilDiv(size_.rows(), block_size_.rows()),
                util::ceilDiv(size_.cols(), block_size_.cols())};
 
   if (size_.isEmpty()) {
-    DLAF_ASSERT(ld_tile_ >= 1, "Invalid Leading Dimension!", ld_tile_);
-    DLAF_ASSERT(tile_offset_row_ >= 1, "Invalid Tile Row Offset!", tile_offset_row);
-    DLAF_ASSERT(tile_offset_col_ >= 1, "Invalid Tile Col Offset!", tile_offset_col);
+    DLAF_ASSERT(ld_tile_ >= 1, ld_tile_);
+    DLAF_ASSERT(tile_offset_row_ >= 1, tile_offset_row);
+    DLAF_ASSERT(tile_offset_col_ >= 1, tile_offset_col);
   }
   else {
     SizeType last_rows = size_.rows() - block_size_.rows() * (nr_tiles_.rows() - 1);
 
     SizeType max_rows_tiles = std::min(size_.rows(), block_size_.rows());
 
-    DLAF_ASSERT(ld_tile_ >= max_rows_tiles, "Invalid Leading Dimension!");
-    DLAF_ASSERT(tile_offset_row_ >= static_cast<std::size_t>(max_rows_tiles),
-                "Invalid Tile Row Offset!");
+    DLAF_ASSERT(ld_tile_ >= max_rows_tiles, ld_tile_, max_rows_tiles);
+    DLAF_ASSERT(tile_offset_row_ >= static_cast<std::size_t>(max_rows_tiles), tile_offset_row,
+                static_cast<std::size_t>(max_rows_tiles));
```

Comment:
```suggestion
    DLAF_ASSERT(tile_offset_row_ >= max_rows_tiles, tile_offset_row,
                max_rows_tiles);
```
`size_t` is not needed anymore here since #247.